packages/datadog-plugin-mongodb-core/test/core.spec.js

```diff
@@ -109,90 +109,6 @@ describe('Plugin', () => {
             }, () => {})
           })
 
-          it('should sanitize the query', done => {
-            agent
-              .use(traces => {
-                const span = traces[0][0]
-                const query = '{"foo":"?","bar":{"baz":"?"}}'
-                const resource = `find test.${collection} ${query}`
-
-                expect(span).to.have.property('resource', resource)
-                expect(span.meta).to.have.property('mongodb.query', query)
-              })
-              .then(done)
-              .catch(done)
-
-            server.command(`test.${collection}`, {
-              find: `test.${collection}`,
-              query: {
-                foo: 1,
-                bar: {
-                  baz: [1, 2, 3]
-                }
-              }
-            }, () => {})
-          })
-
-          it('should sanitize buffers as values and not as objects', done => {
```

Comment:
Some of the tests including this one are still relevant even without obfuscation.